pandas/io/tests/test_date_converters.py

```diff
@@ -138,6 +138,19 @@ def date_parser(date, time):
                                names=['datetime', 'prn']))
         assert_frame_equal(df, df_correct)
 
+    def test_parse_date_column_with_empty_string(self):
+        # GH 6428
+        data = """case,opdate
+                  7,10/18/2006
+                  7,10/18/2008
```

Comment:
hmm, I think we did that on purpose..

that whole clause is bogus, ``errors='ignore'`` means it will *never* raise.

but its a bit tricky, you want to try to parse, then fallback on a softer-parse if necessary.

try playing around with this and see what happens.